multimedia/templates/multimedia/book_list.html

```diff
@@ -0,0 +1,19 @@
+{% extends "multimedia/book_base.html" %}
+{% load staticfiles %}
+
+{% block content %}
+{{ block.super }}
```

Comment:
Yeah I read it alr but still don't understand why you need it here. So it's basically so that you can append more things to the block in the templates that extends from it, right? But why do you need it here?